packages/react-native-aztec/RNTAztecView.podspec

```diff
@@ -9,15 +11,14 @@ Pod::Spec.new do |s|
   s.license          = package['license']
   s.homepage         = package['homepage']
   s.authors          = package['author']
-  s.source            = { :git => 'https://github.com/WordPress/gutenberg.git' }
+  s.source           = { git: 'https://github.com/WordPress/gutenberg.git' }
   s.source_files     = 'ios/RNTAztecView/*.{h,m,swift}'
   s.public_header_files = 'ios/RNTAztecView/*.h'
   s.requires_arc     = true
-  s.platforms        = { :ios => "13.0" }
+  s.platforms        = { ios: '13.0' }
   s.swift_version    = '5.0'
-  s.xcconfig         = {'OTHER_LDFLAGS' => '-lxml2',
-                        'HEADER_SEARCH_PATHS' => '/usr/include/libxml2'}
+  s.xcconfig         = { 'OTHER_LDFLAGS' => '-lxml2',
+                         'HEADER_SEARCH_PATHS' => '/usr/include/libxml2' }
   s.dependency         'React-Core'
-  s.dependency         'WordPress-Aztec-iOS', '~> 1.19.8'
```

Comment:
I'm hesitant to use `~> 1.19` here. I know patch versions shouldn't introduce break changes, but from experience when bumping the Aztec library, we encounter from time to time regressions that need attention. This is mainly caused because we customize and override parts of the library when integrating it into Gutenberg. That said, I'd rather use a fixed version and only update the version when we need it.

Reply:
Fair enough. Thank you for the additional context, @fluiddot . Addressed in de8ddfbb094323fc6e1a0bcf91da2b5e5cc5933e